### ISSUE TITLE
Feature Request: Dogma Units

### ISSUE THREAD

**Issue by blitzmann** [ccpgames/esi-issues#769]
# Feature request

### Route
/dogma/units/
/dogma/units/{unit_id}

### Authentication
None

### Use case
This is available in the SDE, but not in ESI. This information is helpful for fitting programs like pyfa. If pyfa were to get it's data from ESI (which is the eventual goal), it wouldn't be able to switch completely until this endpoint it developed.

It's not a high priority, just noticed that there wasn't an endpoint for it when looking at the attributes endpoint (which references units)

### Example returns
```
{
  "description": "Used to modify damage resistance. Damage resistance bonus.\r\n0.1 = 90%\r\n0.9 = 10%",
  "display_name": "%",
  "unit_id": 111,
  "unit_name": "Inversed Modifier Percent",
}
```

Basically same thing that's in `eveUnits.yaml`

### Previous XML or CREST route
N/A

**Comment by a-tal** [comment]
dupe of #404
